core/src/main/java/io/camunda/migrator/impl/RuntimeValidator.java

```diff
@@ -84,10 +84,15 @@ public void validateProcessInstanceState(String legacyRootProcessInstanceId, Con
    */
   public void validateC7FlowNodes(String processDefinitionId, String activityId) {
     BpmnModelInstance c7BpmnModelInstance = repositoryService.getBpmnModelInstance(processDefinitionId);
+    String multiInstanceBodySuffix = "#multiInstanceBody";
```

Comment:
Nit
We already have a constant for this in `C7Utils.MULTI_INSTANCE_BODY_SUFFIX`

Reply:
Updated